src/libs/elektra/kdb.c

```diff
@@ -1934,14 +1934,18 @@ int kdbGet (KDB * handle, KeySet * ks, Key * parentKey)
 		ELEKTRA_SET_INTERNAL_ERROR (parentKey,
 					    "Couldn't divide keys into mountpoints before poststorage. Please report this bug at "
 					    "https://issues.libelektra.org.");
+		keyDel (defaultCutpoint);
```

Comment:
Wouldn't it be better if you delete `defaultCutpoint` and `defaults` after the label `error`?

Reply:
The problem is `goto error` can already happens before those two vars exist, so it's either doing it like this, or setting them to `NULL` at the beginning and checking for `NULL` after the label. I felt like this is the better solution. Am I missing something?

Reply:
No, it's alright.